pkg/controller/storagecluster/controller_test.go

```diff
@@ -9164,6 +9164,23 @@ func TestShouldPreflightRun(t *testing.T) {
 	require.True(t, preflightShouldRun(cluster))
 	logrus.Infof("vshpere cloud w/PX >= 3.1, preflight will run")
 
+	// TestCase: Vsphere cloud provider with Install mode 'local'
+	logrus.Infof("check vsphere cloud w/local install mode...")
+	cluster.Spec.Image = "portworx/oci-image:3.1.0"
+	env[1].Name = "VSPHERE_INSTALL_MODE"
+	env[1].Value = "local"
+	cluster.Spec.Env = env
+
+	err = preflight.InitPreflightChecker(k8sClient)
+	require.NoError(t, err)
+
+	driver.EXPECT().UpdateDriver(gomock.Any())
+	driver.EXPECT().SetDefaultsOnStorageCluster(gomock.Any())
+	err = controller.setStorageClusterDefaults(cluster)
+	require.NoError(t, err)
+	require.False(t, preflightShouldRun(cluster))
+	logrus.Infof("vshpere cloud w/local install mode will not run")
```

Comment:
nit: debug logs?

Reply:
These are logs added to separate the test output.